hazelcast/src/main/resources/hazelcast-config-5.1.json

```diff
@@ -2940,7 +2940,7 @@
         "migration"
       ]
     },
-    "MemorySize": {
+    "Capacity": {
```

Comment:
this is changing the 5.1 json schema

Reply:
Yep,  by mistake. Reverted: 1135d7cd77ef1592681bdc17a0779a1f4d21915e